arrays/v2/sum.go

```diff
@@ -1,10 +1,10 @@
 package main
 
-// Sum calculates the total from an array of numbers
-func Sum(numbers [5]int) int {
-	sum := 0
-	for _, number := range numbers {
-		sum += number
+// Soma calcula o valor total dos números em um array
```

Comment:
Dúvida: `array` ou `slice`? Sei que por tradução o correto é usar `array` mas acabamos usando o slice da mesma forma que nos demais exemplos. Posso estar viajando também, não descarte essa opção XD

Reply:
Então, a diferença desse exemplo para os outros é que esse de fato é um array. No Go, um array é uma lista com a capacidade definida: `[5]int` nesse caso. Quando não definimos a capacidade da lista, ela é um slice: `[]int`.
[Esse tópico](https://github.com/larien/learn-go-with-tests/blob/traducao/arrays-and-slices/primeiros-passos-com-go/arrays-e-slices.md) fala sobre isso :D mas é um bom ponto: a gente não costuma usar array na prática e esse capítulo também avisa isso hahaha

Reply:
@larien justo! Valeu demais pela explicação!